python/lsst/ctrl/bps/bps_utils.py

```diff
@@ -256,3 +257,46 @@ def _dump_env_info(filename):
         file = file.with_suffix(f"{file.suffix}.yaml")
     with open(file, "w", encoding="utf-8") as fh:
         yaml.dump(dict(os.environ), fh)
+
+
+def _make_id_link(config, run_id):
+    """Make id softlink to the submit run directory.
```

Comment:
In the docstring, I would include information that this function won't create the link if `makeIdLink` is not set to `True` in the config.